### ISSUE TITLE
dex.spec for building a rpm?

### ISSUE THREAD

**Issue by SEJeff** [dexidp/dex#811]
Figured it was worth asking about a spec file for building dex into a rpm. I'm happy to do this if it would be accepted.

I've got a policy of only allowing docker containers (on premise) that come from a RHEL based internal image, and generally still package things as rpm, even dex. Figured it made sense to contribute it back if that seems sensible?

Ditto for a systemd service file if someone might want to run dex standalone on a server.

**Comment by ericchiang** [comment]
> I'm happy to do this if it would be accepted.

Yeah absolutely. It's been a while since I built an RPM, but would be happy to review.

We haven't been doing active development on standalone deployments mostly because we don't use them internally, not because we don't want them.

**Comment by SEJeff** [comment]
Great, once I've got it working, I'll send a PR. I figured you'd be ok with a systemd service file once it is done since I know CoreOS :heart: systemd.

**Comment by nabokihms** [comment]
I will close this one since providing package builds for different Linux distributions has yet to be planned. Feel free to open a new issue or PR if you still need one.
